src/stylesheets/core/_utilities.scss

```diff
@@ -150,7 +152,7 @@
       z-index: 1;
     }
 
-    &.usa-current {
+    &.usa-current { /* stylelint-disable-line selector-no-qualifying-type */
```

Comment:
This rule is not re-enabled.

But I'd also potentially prefer to just move `.usa-current` outside of `a` because I'm not sure the extra specificity is needed?